entrypoint.sh

```diff
@@ -12,6 +12,10 @@ optionals+=""
 [[ -S ${CRI_RUNTIME_ENDPOINT} ]] && optionals+="-cri-runtime-endpoint ${CRI_RUNTIME_ENDPOINT} "
 [[ "${PAUSE_IMAGE}" ]] && optionals+="-pause-image ${PAUSE_IMAGE} "
 [[ "${VXLAN_PORT}" ]] && optionals+="-vxlan-port ${VXLAN_PORT} "
+[[ "${CACERT_FILE}" ]] && optionals+="-ca-cert-file ${CACERT_FILE} "
+[[ "${CERT_FILE}" ]] && optionals+="-cert-file ${CERT_FILE} "
+[[ "${CERT_KEY}" ]] && optionals+="-cert-key ${CERT_KEY} "
```

Comment:
the three of these must be set together no? maybe worth checking it

Reply:
Good point. The cert_file and cert_key are must. And if self-signed certificate is used then cacert_file is also needed. I have added this in doc. I'll update the entrypoint.sh to reflect this.